src/qt/rpcconsole.cpp

```diff
@@ -274,6 +277,19 @@ RPCConsole::RPCConsole(const PlatformStyle *platformStyle, QWidget *parent) :
     connect(ui->fontSmallerButton, SIGNAL(clicked()), this, SLOT(fontSmaller()));
     connect(ui->btnClearTrafficGraph, SIGNAL(clicked()), ui->trafficGraph, SLOT(clear()));
 
+    //Setup autocomplete and attach it
+    QStringList wordList;
+    for (int i =0; i < 53; ++i)
```

Comment:
@MarcoFalke That would require C++11 which i believe at this point is not supported yet.

@jonasschnelli Apparently this is much trickier than it would seem to be. Straightforward way would be to use vector but would be kind of redundant for a const array. Will see what i can do about it.